abel/classes/interstage/plasma_lens/impactx.py

```diff
@@ -31,6 +31,12 @@ class InterstagePlasmaLensImpactX(InterstagePlasmaLens):
         If ``True``, inserts ImpactX ``BeamMonitor`` in the lattice for 
         recording intermediate beam states. Defaults to ``False``.
 
+    enable_isr_on_ref_part : bool, optional
+        Flag for applying `ISR to the reference particle <https://impactx.readthedocs.io/en/latest/usage/python.html#impactx.ImpactX.isr_on_ref_part>`_.
```

Comment:
You can avoid Sphinx "duplicate reference" warnings by making the link "anonymous" with double underscore:
```suggestion
        Flag for applying `ISR to the reference particle <https://impactx.readthedocs.io/en/latest/usage/python.html#impactx.ImpactX.isr_on_ref_part>`__.
```